src/components/Card.jsx

```diff
@@ -0,0 +1,18 @@
+const Card = ({lname, fname, employment, image}) => {
+    return ( 
+        <div className="w-[20rem] h-[25rem] group [perspective: 1000px]  p-5 bg-white rounded-xl shadow-sm  hover:shadow-xl hover:shadow-neutral-700 cursor-pointer shadow-black">
+          <div className="relative [transform-style: preserve-3d] group-hover:[transform:rotateY(180deg)] group-hover:[backface-visibility:hidden] transition-all duration-200">
+            <div className="absolute top-0 left-0 ">
```

Comment:
You can refactor your codes by removing unnecessary spaces
```suggestion
            <div className="absolute top-0 left-0">
```